### ISSUE TITLE
 Removing "fail, error, fatal" verbiage not associated with error in python and C++ tests

### ISSUE THREAD

**Issue by TT-billteng** [tenstorrent/tt-metal#5876]
From CI WTF discussion with @pgkeller

**Comment by vtangTT** [comment]
as per slack discussion: 

`>90% of the cases i'm seeing are from tests that are intentionally trying to trigger an error, so the message printed is the same as if there was actually a problem, gtest then catches it with EXPECT_THROW or something similar
going to ignore these since idt i should be changing those error messages, and gtest currently does not have a way to filter or change verbosity from what i've seen`

**Comment by pgkeller** [comment]
Yeah, this is a hard problem w/o a quick fix - it's architectural/philosophical: should the runtime print errors like this at all?  Or should that be up to the clients?  If errors are trappable/recoverable, the API shouldn't be printing them IMHO.

**Comment by fahmedchanTT** [comment]
@tt-rkim Is this case still relevant? Please close if not.

**Comment by minaliuTT** [comment]
@vtangTT Just checking, is this issue still valid, or can it be closed?

**Comment by tt-rkim** [comment]
Closing as not relevant. Vincent is now in another organization.
